client/components/web-preview/content.jsx

```diff
@@ -279,7 +286,7 @@ export class WebPreviewContent extends Component {
 							ref={ this.setIframeInstance }
 							className="web-preview__frame"
 							src="about:blank"
-							onLoad={ this.setLoaded }
+							onLoad={ this.setIframeLoaded }
```

Comment:
Careful, we're not attaching `this.setLoaded`! Would recommend not attaching directly and have this as part of the error case for `setLoaded`

Reply:
yeh, thanks for the reminder, this was a temporary change to verify the approach worked, lots of stuff happening in this.setLoaded - but there is a bit of a chicken & egg thing going on between that, which is also triggered by the iframe child messaging, and the iframe onload event, so I need to spend a bit of time working out how to handle this without introducing unexpected race conditions, etc.

Reply:
I have merged the iFrame loaded check back into setLoaded now. I just need to try and work out how to best test each of the branches in setLoaded to make sure my additional logic hasn't broken something.